WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1087,7 +1092,10 @@ decisions = {
 		allow = {
 			war = no
 			NOT = { trait = incapable }
-			completely_controls = d_aragon
+			completely_controls = c_alto_aragon
+			completely_controls = c_zaragoza
+			completely_controls = c_albarracin
+			completely_controls = c_calatayud
```

Comment:
This doesn't work, it will allow the AI to actually take the decision. Add an always = no.